packages/workflows/src/handlers/inventory/create-inventory-items.ts

```diff
@@ -28,47 +24,27 @@ export async function createInventoryItems({
     return void 0
   }
 
-  const variants = data.products.reduce(
-    (
-      acc: ProductTypes.ProductVariantDTO[],
-      product: ProductTypes.ProductDTO
-    ) => {
-      return acc.concat(product.variants)
-    },
-    []
-  )
-
   const result = await Promise.all(
-    variants.map(async (variant) => {
-      if (!variant.manage_inventory) {
-        return
-      }
-
-      const inventoryItem = await inventoryService!.createInventoryItem(
-        {
-          sku: variant.sku!,
-          origin_country: variant.origin_country!,
-          hs_code: variant.hs_code!,
-          mid_code: variant.mid_code!,
-          material: variant.material!,
-          weight: variant.weight!,
-          length: variant.length!,
-          height: variant.height!,
-          width: variant.width!,
-        },
-        {
-          transactionManager: (context.transactionManager ??
-            context.manager) as any,
-        }
```

Comment:
@adrien2p - what about here?

Reply:
Here you can remove the propertry transactionManager entirely